### DISCUSSION TITLE
XP's User Interface

### DISCUSSION THREAD

**Discussion by Sleora** [Open-Shell/Open-Shell-Menu#671] [Q&A]
Are there any way to get the full XP's User Interface for all windows (explorer, web browser, etc...) like in this image ? With all the icons, menu and buttons ? 
![EuJ2n9FXUAYdHQY](https://user-images.githubusercontent.com/47717666/112328782-ed69c300-8cb6-11eb-93d5-7cfc904ae435.png)

(By the way, I'm trying to modify the shortcut of the shutdown button to a custom one ("Windows XP Shutdown Menu"). How can I change it ?)

**Comment by bonzibudd** [comment]
You can change the Shutdown command under the “Customize Start menu” tab, along with other items. You could try installing Classic Explorer (part of Open-Shell) to add the XP toolbar to Explorer to get the interface more similar.

**Reply by Sleora** [reply]
Wow, thanks ! Now I have a shutdown button just like Windwos XP ^_^
